plugin/storage/integration/es_index_cleaner_test.go

```diff
@@ -123,6 +125,87 @@ func TestIndexCleaner(t *testing.T) {
 	}
 }
 
+func TestIndexRollover_FailIfILMNotPresent(t *testing.T) {
+	client, err := createESClient()
+	require.NoError(t, err)
+	// make sure ES is clean
+	cleanES(t, client, "jaeger-ilm-policy")
```

Comment:
should `cleanES` always be used with a `defer` to guarantee cleanup even on failure so the ES state is clean for other tests?

Reply:
Added.